Calibration/HcalCalibAlgos/plugins/HcalHBHEMuonAnalyzer.cc

```diff
@@ -497,7 +495,7 @@ void HcalHBHEMuonAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSet
       if (RecMuon.innerTrack().isNonnull()) {
         const reco::Track* pTrack = (RecMuon.innerTrack()).get();
         spr::propagatedTrackID trackID = spr::propagateCALO(pTrack, geo_, bField, (((verbosity_ / 100) % 10 > 0)));
-        if ((RecMuon.p() > pMinMuon_) && (trackID.okHCAL))
+        if ((RecMuon.p() > 10.0) && (trackID.okHCAL))
```

Comment:
this line of change introduces a hardcoded cut value as opposed to the original one... it's better to avoid hardcoded numbers

Reply:
Thanks for pointing this out. I am correcting it